src/components/pages/CurriculumInfo/tabs/UnitsTab/UnitsTab.tsx

```diff
@@ -214,129 +290,245 @@ const UnitsTab: FC<UnitsTabProps> = ({ data }) => {
+                            isSelectedDomain(year, domain) ? "black" : "white"
+                          }
+                          key={domain.tag_id}
+                          label={domain.title}
+                          onClick={() => handleSelectDomain(year, domain)}
+                          size="small"
+                          data-testid="domain-button"
+                        />
+                      ))}
+                    </Box>
+                  )}
+                  {tiers.length > 0 && (
+                    <Box $mb={32}>
+                      {tiers.map((tier) => (
+                        <Button
+                          $mr={16}
+                          key={tier.tier_slug}
+                          label={tier.tier}
+                          onClick={() => handleSelectTier(year, tier)}
+                          size="small"
+                          variant="minimal"
+                          isCurrent={isSelectedTier(year, tier)}
+                          currentStyles={["underline"]}
+                          data-testid="tier-button"
+                        />
+                      ))}
+                    </Box>
+                  )}
+                  <Flex $flexWrap={"wrap"} data-testid="unit-cards">
+                    {units
+                      .filter((unit) => isVisibleUnit(year, unit))
+                      .map((unit, index) => {
+                        const isHighlighted = isHighlightedUnit(unit);
+                        return (
+                          <Card
+                            key={unit.slug}
+                            $background={isHighlighted ? "black" : "white"}
+                            $color={isHighlighted ? "white" : "black"}
+                            $flexGrow={"unset"}
+                            $mb={32}
+                            $mr={28}
+                            $pb={64}
+                            $position={"relative"}
+                            $width={[
+                              "100%",
+                              "calc(50% - 28px)",
+                              "calc(33% - 26px)",
+                            ]}
+                            data-testid={
+                              isHighlighted
+                                ? "highlighted-unit-card"
+                                : "unit-card"
+                            }
                           >
-                            <OakLink
-                              page="lesson-index"
-                              viewType="teachers-2023"
-                              programmeSlug={buildProgrammeSlug(unit)}
-                              unitSlug={unit.slug}
-                              data-testid="unit-link"
+                            <BrushBorders
+                              color={isHighlighted ? "black" : "white"}
+                            />
+                            <OutlineHeading tag={"div"} $fontSize={24} $mb={12}>
```

Comment:
This looks like it's a pre-existing issue, but from what I can see in Figma these unit numbers should be 600 font weight instead of 300 (`Heading/Regular/Heading 5`).